pages/torchbox-academy.tsx

```diff
@@ -13,16 +12,20 @@ type TorchboxAcademyPageProps = {
 const TorchboxAcademyPage: NextPage<TorchboxAcademyPageProps> = ({
     preview,
     content,
-}) => (
-    <Layout theme="INDIGO" preview={preview} jobsAvailable={8}>
-        <h1>Torchbox Academy</h1>
-        <RichText theme="INDIGO" content={content.heroSubtitle} />
-        <Image
-            src={content.heroImage.url}
-            alt={content.heroImage.description}
-        />
-    </Layout>
-);
+}) => {
+    const academyCollectionItem = content.itemsCollection.items.find(
+        (obj: any) => obj.__typename === 'Academies',
```

Comment:
**Suggestion**: I wonder if we can import or create a generic type for this?

Reply:
Nice, I've created a type that imports the Academy type :)